bedrock/mozorg/templates/mozorg/about/policy/transparency/jan-dec-2015.html

```diff
@@ -0,0 +1,247 @@
+{# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/. -#}
+
+{% extends "mozorg/about/policy/transparency/report-base.html" %}
+
+{% block page_title %}{{ _('Transparency Report - 2015') }}{% endblock %}
+
+{% block report_title %}
+  <h1>{{ _('Transparency Report') }} <span>{{ _('Reporting Period: January 1, 2015 to December 31, 2015') }}</span></h1>
+{% endblock %}
+
+{% block report_body %}
+  <section class="section" id="government-user-data">
+    <h2 class="section-title" data-accordion-role="tab">{{ _('Government Demands for User Data') }}</h2>
+
+    <div class="content" data-accordion-role="tabpanel">
+      <p>
+        {{ _('In the Reporting Period, Mozilla did not receive any government demands through Legal Processes for user data.') }}
+      </p>
+
+      <table class="table">
+        <thead>
+          <tr>
+            <th scope="col">{{ _('Legal Processes') }}</th>
+            <th scope="col">{{ _('Received') }}</th>
+            <th scope="col">{{ _('Data Produced') }}</th>
+          </tr>
+        </thead>
+        <tbody>
+          <tr>
+            <th scope="row">
+              <a href="{{ url('mozorg.about.policy.transparency.index') + '#dfn-search-warrant' }}">{{ _('Search Warrants') }}</a>
+            </th>
+            <td>0</td>
+            <td>N/A</td>
```

Comment:
I wasn't sure about N/A... it's an abbreviation of English words, but is also a sort of programmatic symbol. Maybe best to wrap it just in case. And 0 is just 0, I don't think we'd translate any other numerals either.